uint16 robot_ID
string state
